src/plugins/lua/rbl.lua

```diff
@@ -63,14 +63,16 @@ local function rbl_cb (task)
   for k,rbl in pairs(rbls) do
 
     (function()
-      if not notgot['user'] and not havegot['user'] then
-	havegot['user'] = task:get_user()
-	if havegot['user'] == nil then
-	  notgot['user'] = true
-	end
-      end
-      if havegot['user'] ~= nil and rbl['user'] == false then
-	return
+      if rbl['user'] == false then
```

Comment:
Yes, 'user' and 'default_user' answer the question: 'should I use this RBL if the user is authenticated?' so to activate the exclusion it is set to false. ;)